src/hotspot/cpu/aarch64/gc/g1/g1BarrierSetAssembler_aarch64.hpp

```diff
@@ -69,7 +70,7 @@ class G1BarrierSetAssembler: public ModRefBarrierSetAssembler {
 #endif
 
   void load_at(MacroAssembler* masm, DecoratorSet decorators, BasicType type,
-               Register dst, Address src, Register tmp1, Register tmp_thread);
+               Register dst, Address src, Register tmp1, Register tmp2);
```

Comment:
Need similar modification for function definition in file barrierSetAssembler_aarch64.cpp

Reply:
👍 . Fixed